aldryn_events/tests/frontend/integration/specs/spec.events.crud.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global describe, it, browser */
+
+// #############################################################################
+// INTEGRATION TEST
+describe('Aldryn Events tests: ', function () {
+    var eventsPage = require('../pages/page.events.crud.js');
```

Comment:
can we move it out of describe?
requires in general should be on top of the file
we are going to go into the ecmascript 6 land and and reality is that possibility of this behaviour would go away soon so it's better to be consistent